primitiv/eigen_device.cc

```diff
@@ -13,6 +13,15 @@
 #include <cmath>
 #include <iostream>
 
+// NOTE(vbkaisetsu)
+// Eigen contains a few LGPL-licensed features. It will conflict with some
+// non-free licensed software that can link statically with Apache licensed
+// code.
+// EIGEN_MPL2_ONLY guarantees that primitiv does not use LGPL-licensed feature.
+//
+// For more ditails, see:
+// http://eigen.tuxfamily.org/index.php?title=Main_Page#License
+#define EIGEN_MPL2_ONLY
```

Comment:
I guess this option is necessary only when PRIMITIV_BUILD_STATIC_LIBRARY is on. But it may be good to always enable this option to prevent some accidental linkages between primitiv and Eigen. Could you put your opinion?